src/Messages.js

```diff
@@ -1182,15 +1182,10 @@ export default defineMessages({
     kebabResumeAnalysis: {
         id: 'kebab.resumeAnalysis',
         description: 'Kebab item',
-        defaultMessage: 'Resume analysis for system'
-    },
-    kebabExcludeAnalysis: {
-        id: 'kebab.excludeAnalysis',
-        description: 'Kebab item',
-        defaultMessage: 'Exclude system from analysis'
+        defaultMessage: 'Resume vulnerability analysis for system'
```

Comment:
I think system needs to be plural
Resume vulnerability analysis for system**s**

Reply:
I think we are using this string for both table row action (where it should be singular) and table toolbar bulk action (where it should be singular for 1 system selected and plural for more). Ideal solution would be to use intl pluralization, count = 1 for row action and count = selectedSystems array length for table toolbar.

Reply:
@leSamo could you provide an example cause @tkasparek  might not be familiar with it

Reply:
If you want to format a message depending on context in react intl you use formats. One useful format which we use a lot is plural format, which allows you to pluralise words, for example:
Business risk updated for 1 **CVE**
Business risk updated for 2 **CVEs**

In Messages.js it looks like this:
`Business risk updated for {count, plural, one {# CVE} other {# CVEs}}`

Format is wrapped in { } and has 3 parts:
`count` - name of variable you pass from code
`plural` - name of format
the rest specifies substring depending on how much `count` is.

You have to pass the variable to the message as a property of object in a second argument in intl.formatMessage:
`intl.formatMessage(messages.businessRiskUpdateSuccessful, { count: cveList.length || 0 })`

Documentation link: https://formatjs.io/docs/core-concepts/icu-syntax#plural-format